code/modules/heavy_vehicle/interface/screen_objects.dm

```diff
@@ -217,7 +221,8 @@
 	owner.maintenance_protocols = toggled
 	var/main_color = owner.maintenance_protocols ? "#d1d1d1" : "#525252"
 	maptext = "<span style=\"font-family: 'Small Fonts'; color: [main_color]; -dm-text-outline: 1 #242424; font-size: 6px;\">MAINT</span>"
-	to_chat(usr, "<span class='notice'>Maintenance protocols [owner.maintenance_protocols ? "enabled" : "disabled"].</span>")
+	if(usr.loc == owner)
+		to_chat(usr, "<span class='notice'>Maintenance protocols [owner.maintenance_protocols ? "enabled" : "disabled"].</span>")
```

Comment:
This loc check being copypasted everywhere isn't really... appropriate. You should shunt this stuff to a proc on /obj/screen/mecha such as `notify_user(var/text)`, do the checks and to_chat there.